docs/api/sql/Function.md

```diff
@@ -1552,18 +1552,18 @@ SELECT ST_AsText(ST_PointOnSurface(ST_GeomFromText('LINESTRING(0 5 1, 0 0 1, 0 1
 
 ```
 
-## ST_PrecisionReduce
+## ST_ReducePrecision
 
 Introduction: Reduce the decimals places in the coordinates of the geometry to the given number of decimal places. The last decimal place will be rounded.
 
-Format: `ST_PrecisionReduce (A:geometry, B:int)`
```

Comment:
Please add one more sentence saying that the name of this function was called ST_PrecisionReduce in versions prior to v1.5.0.

Reply:
I just noticed that `ST_ReducePrecision(geometry g, float8 gridsize);` in postgis needs float as the second param. That means use '0.1' as 1, '0.01' as 2. Should we follow the function in postgis?

Reply:
em... I think the Sedona implementation of this function is very different from PostGIS. I suggest that, in this PR, we don't adopt the PostGIS implementation. We will address this in a different PR.

Reply:
Yeah, I think so. Here we can keep Sedona implementation. Should we use the keep the function name the same as PostGIS like 'ST_ReducePrecision '? Or we can revert to the old name 'ST_PrecisionReduce ', which doesn't exsit in PostGIS.

Reply:
@yyy1000 I think changing to `ST_ReducePrecision` is fine. No need to revert it back. Just update the doc then we are good to go.